.github/workflows/patch-account-ai-settings.yml

```
@@ -0,0 +1,103 @@
+          from pathlib import Path
+
+          path = Path("src/routes/_app.tsx")
+          text = path.read_text()
+
+          import_needle = 'import { AccountStatus } from "@/features/auth/AccountStatus";\nimport { useAuth } from "@/features/auth/use-auth";'
+          import_replacement = 'import { AccountStatus } from "@/features/auth/AccountStatus";\nimport { useAuth } from "@/features/auth/use-auth";\nimport { useAiDelegation } from "@/features/ai/use-ai-delegation";'
+          assert text.count(import_needle) == 1, "expected AccountStatus/useAuth import block once"
+          text = text.replace(import_needle, import_replacement, 1)
+
+          callback_needle = '''  const applyPrivateData = useCallback((payload: PrivateDataPayloadV1) => {
+    latestMeetings.current = payload.schedule;
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)